webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala

```diff
@@ -528,12 +644,22 @@ class NodeGroupForm(
           )
         }
 
-        popup.popupWarningMessages match {
-          case None    =>
-            popup.onSubmit()
-          case Some(_) =>
-            SetHtml("confirmUpdateActionDialog", popup.popupContent()) &
-            JsRaw("""createPopup("confirmUpdateActionDialog")""")
+        if (popup.popupWarningMessages.isEmpty && dependingSubgroups.isEmpty) {
+          popup.onSubmit()
+        } else {
+          val html: NodeSeq = dependingSubgroups match {
+            case None      => popup.popupContent()
+            case Some(msg) =>
+              val cssSel: CssSel = "#explanationMessageZone *+" #>
```

Comment:
that cssSel means: "add after the last child in content"

Reply:
woaaah it's quite inventive